### ISSUE TITLE
paper writing and bound books issues with colors and broken symbols

### ISSUE THREAD

**Issue by redpepperphoenix** [Skyrat-SS13/Skyrat-tg#1621]
## Round ID: 
[16](https://scrubby.melonmesa.com/round/16)

## Reproduction: 
There are a few issues regarding paper writing and binding books.
First, editing papers with a black pen makes them blend in as white letters, can be read in the preview, and read normally. 
When the paper is bound as a book, characters like " are replaced with â€œâ€¦ and the words are also in white, making reading the book impossible without highlighting it.
Different color pens can be seen editing and on books but show as black letters instead of their color of red or blue.
in summary: 
-black pens are white while editing and on bound books, 
-Some symbols break when binding books, 
-preview/reading papers only show black letters. making different color pens black. can see red and blue on edit and on books

Full rejoin doesn't fix the issue

**Comment by tf-4** [comment]
This is ooooold and paper has been changed significantly more than once. I don't think this is still an issue, but let me know if it is.
